scripts/fvt-tools.sh

```diff
@@ -9,23 +9,50 @@ fvtlog() {
   echo "$(date +"%F %T %Z")" "[fvt]" "$*"
 }
 
-list_nodes() {
-  kubectl get nodes --no-headers -o jsonpath='{.items[*].metadata.name}'
+update_node_list() {
+  mapfile -d' ' -t NODES < <(kubectl get nodes --no-headers -o jsonpath='{.items[*].metadata.name}')
 }
 
-list_pods() {
-  kubectl get pods -A --no-headers --selector=name=staticroute-operator -o wide | awk '{print $2}'
+pick_non_master_node() {
+  for index in ${!NODES[*]}
+  do
+    kubectl get no "${NODES[$index]}" --show-labels | grep 'node-role.kubernetes.io/master=' > /dev/null && continue || echo -ne "${NODES[$index]}"; break
+  done
+}
+
+create_hostnet_pods() {
+  for index in ${!NODES[*]}
+  do
+    kubectl run --generator=run-pod/v1 hostnet-"${NODES[$index]}" --labels="fvt-helper=hostnet" --overrides="{\"apiVersion\": \"v1\", \"spec\": {\"hostNetwork\":true, \"nodeSelector\": { \"kubernetes.io/hostname\": \"${NODES[$index]}\" }, \"tolerations\": [{ \"operator\": \"Exists\" }]}}" --image busybox -- /bin/tail -f /dev/null
+  done
+  for _ in $(seq ${SLEEP_COUNT}); do
+    actual=$(kubectl get pods --selector=fvt-helper=hostnet --field-selector=status.phase=Running --no-headers | wc -l)
+    expected=$(kubectl get pods --selector=fvt-helper=hostnet --no-headers | wc -l)
+    fvtlog "Waiting for hostnet helper pods to come up. Actual: ${actual}, expected: ${expected}"
+    if [[ "${actual}" -eq "${expected}" ]]; then
+      break
+    fi
+    sleep ${SLEEP_WAIT_SECONDS}
+  done
 }
 
-get_sr_pod_ns() {
-  kubectl get pods -A --no-headers --selector=name=staticroute-operator -o wide | awk 'NR==1{print $1}'
+delete_hostnet_pods() {
+  kubectl delete po --selector fvt-helper=hostnet
+}
+
+# Function to execute a command on the host network of a node, selected by a pod that is running on it
+# Parameters:
+# - Node name
+# - Command (may be multiple string)
+exec_in_hostnet_of_node() {
+  nodename=$1
```

Comment:
Could you please add the word `local` to this variable?

Reply:
Yep